frontend/src/metabase/entities/data-app-collections.js

```diff
@@ -37,7 +37,7 @@ const DataAppCollections = createEntity({
   },
 
   objectSelectors: {
-    getIcon: () => ({ name: "star" }),
```

Comment:
@kulyk Any reason why we shouldn't just be calling "getDataAppIcon" from `frontend/src/metabase/entities/data-apps/utils.ts`  everywhere. Wasn't that sort of the intended purpose of moving that to a util fn?

Reply:
Cool yeah not a blocker for sure, just wanted to call it out.